lib/osf-components/addon/components/search-help-modal/component.ts

```diff
@@ -0,0 +1,40 @@
+import { layout } from '@ember-decorators/component';
```

Comment:
looks like there's already a search-help-modal in `app-components`, seems like they should be united